.ci/test_r_package_windows.ps1

```diff
@@ -0,0 +1,108 @@
+# Download a file and retry upon failure. This looks like
+# an infinite loop but CI-level timeouts will kill it
+function Download-File-With-Retries {
+  param(
+    [string]$url,
+    [string]$destfile
+  )
+  do {
+    Write-Output "Downloading ${url}"
+    sleep 5;
+    (New-Object System.Net.WebClient).DownloadFile($url, $destfile)
+  } while(!$?);
+}
+
+$env:R_WINDOWS_VERSION = "3.6.3"
+$env:R_LIB_PATH = "$env:BUILD_SOURCESDIRECTORY/RLibrary" -replace '[\\]', '/'
```

Comment:
Hmm, R tools cannot work with Win paths?! Interesting...

Reply:
I was really surprised, but the builds started breaking after I changed from `C:/Rlibrary` to `BUILD_SOURCESDIRECTORY`, and the only difference was that `BUILD_SOUORCEDIRECTORY` included backslashes.